std/variant.d

```diff
@@ -75,17 +75,21 @@ import std.meta, std.traits, std.typecons;
 /++
     Gives the `sizeof` the largest type given.
   +/
-template maxSize(T...)
+template maxSize(Ts...)
 {
-    static if (T.length == 1)
+    align(1) union Impl
     {
-        enum size_t maxSize = T[0].sizeof;
-    }
-    else
-    {
-        import std.algorithm.comparison : max;
-        enum size_t maxSize = max(T[0].sizeof, maxSize!(T[1 .. $]));
+        // See_Also: https://forum.dlang.org/thread/wbpnncxepehgcswhuazl@forum.dlang.org?page=1
+        static foreach (T; Ts)
+        {
+            static if (!is(T == void))
+            {
+                pragma(msg, T.stringof ~ " _store" ~ T.mangleof ~ ";");
+                mixin(T.stringof ~ " _store" ~ T.mangleof ~ ";");
```

Comment:
you can make this
```suggestion
                mixin("T _store" ~ T.mangleof ~ ";");
```

Reply:
Note that `T.mangleof` does not guarantee uniqueness, e.g., `maxSizeOf!(int,int,float)`.

Reply:
Done.